vtStorAtaProtocol/Platform/Linux/ProtocolAtaPassThrough.h

```diff
@@ -34,7 +34,7 @@ class VT_STOR_ATA_PROTOCOL_API cAtaPassThrough : public cProtocolInterface
 {
 
 public:
-    //virtual eErrorCode IssueCommand( const DeviceHandle& Handle, std::shared_ptr<const cBufferInterface> Essense, std::shared_ptr<cBufferInterface> DataBuffer ) override;
+    virtual eErrorCode IssueCommand( const DeviceHandle& Handle, std::shared_ptr<const cBufferInterface> Essense, std::shared_ptr<cBufferInterface> DataBuffer ) override;

```

Comment:
How about if this pull request is merged? It will contain this uncompleted code, right?
Please create difference branch for this task. We should have unique branch for a task.